Resources/Prototypes/Research/civilianservices.yml

```diff
@@ -14,7 +14,7 @@
   - BorgModuleHarvesting
   - SeedExtractorMachineCircuitboard
   - HydroponicsTrayMachineCircuitboard
-
+  - Plantanalyzer
 - type: technology
```

Comment:
Blank line between the prototypes

Reply:
Github is just displaying the change weird. There is no blank line.

Reply:
No I'm saying there **should** be a blank line. Prototypes in yaml should always be separated by a blank line.